services/task-svc/internal/task-template/task_template.go

```diff
@@ -88,3 +88,47 @@ func (ServiceServer) CreateTaskTemplate(ctx context.Context, req *pb.CreateTaskT
 		Id: taskTemplate.ID.String(),
 	}, nil
 }
+
+func (ServiceServer) UpdateTaskTemplate(ctx context.Context, req *pb.UpdateTaskTemplateRequest) (*pb.UpdateTaskTemplateResponse, error) {
+	log := zlog.Ctx(ctx)
+	db := hwgorm.GetDB(ctx)
+
+	// TODO: Auth
+
+	id, err := uuid.Parse(req.Id)
+	if err != nil {
+		return nil, status.Error(codes.InvalidArgument, err.Error())
+	}
+
+	taskTemplate := TaskTemplate{ID: id}
+	updates := req.UpdatesMap()
+
+	if err := db.Model(&taskTemplate).Updates(updates).Error; err != nil {
+		log.Warn().Err(err).Msg("database error")
```

Comment:
not needed

Reply:
what do you mean? the warn message is basically used in every service, should we omit it from now on?

Reply:
Probably a result of copy-pasting. It's not a problem (just double-log), but it's logged here already: https://github.com/helpwave/services/blob/main/libs/common/grpc.go#L258